msgraph-mail/php/composer.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "microsoft/kiota-samples",
   "require": {
-    "microsoft/kiota-abstractions": "^0.8.0",
-    "microsoft/kiota-http-guzzle": "^0.8.2",
-    "microsoft/kiota-serialization-text": "^0.7.0",
-    "microsoft/kiota-serialization-json": "^0.6.0",
-    "microsoft/kiota-authentication-phpleague": "^0.8.2"
+    "microsoft/kiota-abstractions": "dev-fix/promises-generics",
```

Comment:
this will eventually need to be reverted

Reply:
Bumped to stable